test/e2e/util-test.js

```diff
@@ -1,3 +1,9 @@
+const fs = require("fs");
+const path = require("path");
+const serverUrl = require("../../config/playwright.config.js").url;
+
+const dbPath = "./../../data/playwright-test/kuma.db";
```

Comment:
This might sound pedantic, but I don't think that works on windows.
Since likely at some point another dev will try to run the test cases on windows, could we change this to be platform independent?

Reply:
I'm developing on Windows, so can confirm it works. `path.resolve` normalizes the separators internally.

Reply:
ah, good to know 👍🏻